cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java

```diff
@@ -91,6 +93,20 @@ public static void main(String[] args) {
         }
     }
 
+    private static List<String> parseTestSuites(ApplicationArgs appArgs) {
+        boolean runDefaultTests = true;
+        List<String> testSuites = ListUtils.defaultIfNull(appArgs.getTestSuites(), new ArrayList<String>());
+        for(String testSuite : testSuites){
+            if(testSuite.equalsIgnoreCase(NOT_TS + DEFAULT_TESTS)){
+                runDefaultTests = false;
+            }
+        }
+        if(runDefaultTests && !testSuites.contains(DEFAULT_TESTS)){
```

Comment:
What do you think should happen? Currently it will be treated as running the default test suite